### ISSUE TITLE
OMV6 Performance Statistics Not Working

### ISSUE THREAD

**Issue by epicfail83** [openmediavault/openmediavault#1217]
Pls see this thread in the forum:

https://forum.openmediavault.org/index.php?thread/42038-omv6-performance-statistics-not-working/

**Comment by votdev** [comment]
I know you opened some issues before, but i don't want to search for them to get the specs of your system. Which version? Is this a new setup? Migrated from OMV5 to 6? Any hint how to reproduce it?

**Comment by votdev** [comment]
Have you tried to run `omv-salt deploy run rrdcached`?

**Comment by epicfail83** [comment]
Oh sorry for that. I have an Asrock Q1900 ITX Board with 4Gb of RAM. Newest OMV6 Version as a fresh install because of the other problems i had with the upgrade from 5 to 6.
No hint to reproduce it. Just did a fresh install and this is the only thing not working. 
My bootlog is full with those errors:
`collectd[2884]: rrdcached plugin: Failed to connect to RRDCacheD at unix:/run/rrdcached.sock: Unable to connect to rrdcached: No such file or directory (status=2)`

I´ll attach the output of the command you suggested. Now all the graphs seem to work. Do i need to execute this command after every reboot?
[output.txt](https://github.com/openmediavault/openmediavault/files/8078708/output.txt)

**Comment by votdev** [comment]
> Do i need to execute this command after every reboot?

No, this shouldn't be the case.

Do you have installed any OMV plugins?

**Comment by epicfail83** [comment]
Yes i have the following installed:

- openmediavault-autoshutdown 6.0.7
- openmediavault-backup 6.0.3
- openmediavault-diskclone 6.0
- openmediavault-diskstats 6.0-5
- openmediavault-kernel 6.1.2
- openmediavault-omvextrasorg 6.0.7
- openmediavault-resetperms 6.0
- openmediavault-sharerootfs 6.0-3
- openmediavault-wakealarm 6.0.1

Error messages in logs seem to have disappeared now after the command from above. Don´t rebootet the NAS.

**Comment by votdev** [comment]
Please report back after you've rebooted the system. If that works we can close the issue because i have no idea what was the root cause of this issue.

**Comment by epicfail83** [comment]
Reboot done. I think this command omv-salt deploy run rrdcached fixed it. Can it?
Everything seems working proberly. No more errors in logs.

**Comment by votdev** [comment]
> Can it?

Sure. It seems the configuration was broken somehow.

**Comment by sheetcam** [comment]
I'm seeing the same issue. Fresh install of openmediavault_6.0.24-amd64.iso.
Installed updates. For some reason the kernel update was held so had to apt upgrade to install it.
When I run omv-mkrrdgraph I just get a whole screen full of these errors:
`
ERROR: Unable to connect to rrdcached: No such file or directory
ERROR: Unable to connect to rrdcached: No such file or directory
`
I tried `omv-salt deploy run rrdcached` and rebooted with the same results. Tried reinstalling with the same results.
`/$ ls -alh /var/lib/rrdcached/db/localhost/
total 56K
drwxr-xr-x 14 root root 4.0K Jul 21 21:46 .
drwxr-xr-x  3 root root 4.0K Jul 21 20:07 ..
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 cpu-0
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 cpu-1
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 cpu-2
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 cpu-3
drwxr-xr-x  2 root root 4.0K Jul 21 21:33 df-pool
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 df-root
drwxr-xr-x  2 root root 4.0K Jul 21 21:46 df-srv-dev-disk-by-uuid-28818abb-d30a-4aba-bf80-4b7b7b4146c3
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 interface-enp2s0
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 interface-lo
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 load
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 memory
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 uptime
`
`ls -alh /var/lib/rrdcached/db/localhost/cpu-0/
total 1.2M
drwxr-xr-x  2 root root 4.0K Jul 21 20:07 .
drwxr-xr-x 14 root root 4.0K Jul 21 21:46 ..
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-idle.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-interrupt.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-nice.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-softirq.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-steal.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-system.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-user.rrd
-rw-r--r--  1 root root 146K Jul 21 20:07 cpu-wait.rrd
`
